sbt-dump-event-schema/src/main/scala/org/virtuslab/ash/DumpEventSchema.scala

```diff
@@ -24,7 +24,8 @@ object DumpEventSchema extends DefaultJsonProtocol {
     }
 
     for {
-      writer <- outputFile.clear().newFileOutputStream().printWriter().autoClosed
+      writer <-
```

Comment:
Whoops... I've tried this implementation, and even when I specify an absolute path, the target file is located under:

`./services/target/home/plipski/hydra-backend/target/foo.json`

instead of:

`/home/plipski/hydra-backend/target/foo.json`

:/

Reply:
So it looks like the current impl treats absolute paths as relative ones :/